### ISSUE TITLE
Alerta de vulnerabilidade

### ISSUE THREAD

**Issue by fabriciofx** [ifpe-cpin/maisfono#57]
O GitHub está avisando que há uma falha de segurança de algum pacote no repositório. Devemos investigar com urgência e resolver esse problema.

**Comment by VictorCardoso22** [comment]
Recebi esse email, mas acredito que atualizar o bootstrap poderá quebrar o layout do sistema.

![captura de tela de 2019-01-19 11-16-13](https://user-images.githubusercontent.com/16945411/51427887-a914a100-1bdb-11e9-9c5f-ab06bd7a90f2.png)

**Comment by fabriciofx** [comment]
@VictorCardoso22 Tem que testar, né? E, mesmo se quebrar, avaliar se consertar é muito difícil, pq se a banca ver isso (e eles vão ver), com certeza será a primeira coisa que vão questionar, certo?

**Comment by VictorCardoso22** [comment]
Entendo, vou verificar.

**Comment by Wylianne** [comment]
@fabriciofx  o @VictorCardoso22 tem razão o que estava deixando o layout estranho era a versão do bootstrap. Infelizmente só encontrei essa versão que atende ao tema que escolhemos.

**Comment by fabriciofx** [comment]
@Wylianne @VictorCardoso22 Certo, mas a pergunta boa é: se mudar a versão do bootstrap vai ter que mudar muita coisa ou pouca coisa?

**Comment by VictorCardoso22** [comment]
@fabriciofx @Wylianne, atualizei o bootstrap e, como foi falado, desconfigurou o visual, principalmente nas tabelas de usuário e fonoaudiólogos. 

@fabriciofx, os arquivos css são cdn, assim dificulta uma possível correção. Mexi diretamente nas tabelas, mas ainda assim não fica legal. 

Criei um branch com essas modificações - [https://github.com/ifpe-cpin/maisfono/tree/lista_pacientes](url)

**Comment by fabriciofx** [comment]
@VictorCardoso22 Estar na CDN não é problema. Basta baixar a versão estática e colocar junto com o site. Mas enfim, melhor focar em colocar as coisas para funcionar, certo?
